src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let numbers = 0;
+
+  return function adder(num) {
+    if (typeof num === 'number') {
+      numbers += num;
+
+      return adder;
+    } else {
```

Comment:
else is redundant here
if code inside `if` statement will be executed, your function will be ended with return statement

Reply:
Thanks for the comment, removing it right away!